easybuild/easyconfigs/s/smooth-topk/smooth-topk-1.0-20210817-foss-2021a.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = "PythonBundle"
+
+name = "smooth-topk"
+version = "1.0-20210817"
+local_commit = "12c1645f187e2fa0c05f47bf1fe48864d4bd2707"
+
+homepage = "https://github.com/oval-group/smooth-topk"
+description = "Smooth Loss Functions for Deep Top-k Classification"
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('tqdm', '4.61.2'),
+    ('PyTorch', '1.10.0'),
+    ('SciPy-bundle', '2021.05'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('future', '0.18.2', {
+        'checksums': ['b1bead90b70cf6ec3f0710ae53a525360fa360d306a86583adc6bf83a4db537d'],
+    }),
+    ('logger', '1.4', {
+        'checksums': ['4ecac57133c6376fa215f0fe6b4dc4d60e4d1ad8be005cab4e8a702df682f8b3'],
+    }),
+    ('torchvision', '0.2.0', {
```

Comment:
We have separate easyconfigs for `torchvision`, should we use a proper dependency here?

`torchvision` 0.2.0 is quite old though, did you try using a more recent version (like 0.11.1 for which an easyconfig is already there)?